scylla/src/transport/connection.rs

```diff
@@ -514,7 +519,36 @@ pub async fn open_named_connection(
     let result = connection.startup(options).await?;
     match result {
         Response::Ready => {}
-        Response::Authenticate => unimplemented!("Authentication is not yet implemented"),
+        Response::Authenticate(authenticate) => {
```

Comment:
We should return an error immediately if username and password is not configured. Now, we will send an invalid `AUTHENTICATE_RESPONSE` message (will contain a single `0` byte).

Reply:
Done! I was a bit confused because of cdrs implementation!